src/styles/blocks/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: Roboto, sans-serif;
+  border: 1px solid $background-color;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 32 19 40;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 16 4;
+    color: $main-accent;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 0 84 0 16;
+    color: $secondary-accent;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16 16 24;
+  }
+
+  &__review-text {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $main-accent;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 16 16;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $secondary-accent;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $main-accent;
+  }
+
+  &__buy-link {
+    background-color: $blue-accent;
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    margin: 0 auto;
+    box-sizing: border-box;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-transform: uppercase;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    color: $background-color;
+
+    &:hover {
+      background-color: $background-color;
+      border: 1px solid $blue-accent;
```

Comment:
Better add a transparent border for the default state of the element and change its color on hover